lib/portlayer/storage/data_source.go

```diff
@@ -53,6 +54,46 @@ func (m *MountDataSource) Source() interface{} {
 }
 
 // Export reads data from the associated data source and returns it as a tar archive
+func (m *MountDataSource) Stat(op trace.Operation, spec *archive.FilterSpec) (*FileStat, error) {
```

Comment:
Ok. Should I return 422 then if there's more than one path in filterspec Inclusions?